packages/gotrue/lib/src/types/types.dart

```diff
@@ -29,6 +29,7 @@ enum OAuthProvider {
   spotify,
   twitch,
   twitter,
+  twitter,
```

Comment:
This should be tiktok, right?

Reply:
@dshukertjr sorry, committing wrong changes